modules/users/client/controllers/password.client.controller.js

```diff
@@ -55,13 +53,13 @@
     function onRequestPasswordResetSuccess(response) {
       // Show user success message and clear form
       vm.credentials = null;
-      vm.success = response.message;
+      Notification.success({ message: response.message, title: 'Password request successful!' });
```

Comment:
@mleanos That text is just the title of the error notification. The actual message from the server would be the message like(which is already in the server controllers):
`An email has been sent to the provided email with further instructions.`
`No account with that username has been found`
So the title would need to be generic to what the request is for right?